src/Mono.Android/Android.Runtime/AndroidRuntimeInternal.cs

```diff
@@ -6,26 +6,12 @@ namespace Android.Runtime
 {
 	public static class AndroidRuntimeInternal
 	{
-		internal static MethodInfo? mono_unhandled_exception_method = null;
 		internal static Action<Exception> mono_unhandled_exception = RuntimeNativeMethods.monodroid_debugger_unhandled_exception;
```

Comment:
Maybe mark this `readonly` while we're at it? Nothing should try to set this `mono_unhandled_exception` field later.